designs/login/index.md

```diff
@@ -0,0 +1,27 @@
+---
+has_children: true
+nav_order: 2
+parent: Home
+layout: default
+title: Login
+has_toc: false
+---
+
+# Login Screen
+
+Unauthenticated users will be redirected to the login screen.
+
+Direct authentication with Kerberos token should be available.
```

Comment:
My first idea would be the automagic login. What is the advantages you see for the previous confirmation?

The kerberos button would just open a small hint telling the user that he should config in his side to make it work, it would not lead to another screen.

Reply:
> My first idea would be the automagic login. What is the advantages you see for the previous confirmation?

Visibility, user knows that is going to log in with that account. And acknowledgment, user can opt out and log in with another account, if they choose so.